CITATION.cff

```diff
@@ -0,0 +1,34 @@
+# CITATION.cff: Citation File Format for reproducible research software
+
+cff-version: 1.2.0
+title: Slither
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Josselin Feist
+    family-names: Feist
+  - given-names: Gustavo
+    family-names: Grieco
+  - given-names: Alex
+    family-names: Grace
+identifiers:
+  - type: doi
+    value: 10.48550/arXiv.1908.09878
+    description: arXiv.1908.09878
```

Comment:
I noticed there's a separate DOI for the IEEE publication, `10.1109/wetseb.2019.00008`, maybe it should be this one instead of/in addition to the arXiv one?

Reply:
IEEE typically has a paywall, can add the link regardless